packages/react-components/src/components/CameraButton.tsx

```diff
@@ -248,6 +250,14 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
     }
   };
 
+  const splitButtonMenuProps: IButtonProps = useMemo(
+    () => ({
+      ...props.splitButtonMenuProps,
+      className: 'camera-split-button'
```

Comment:
swap these two lines around, otherwise we overwright the className passed in (and we are only using the className for ui test)